<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logic Gate</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="resizeAnchor">
       <enum>QGraphicsView::NoAnchor</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionMain_Menu"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuResolution">
     <property name="title">
      <string>Resolution</string>
     </property>
     <addaction name="action600x480"/>
     <addaction name="action1280x720"/>
     <addaction name="action1366x720"/>
     <addaction name="action1920x1080"/>
     <addaction name="actionFullscreen"/>
    </widget>
    <addaction name="menuResolution"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="action600x480">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>640x480</string>
   </property>
  </action>
  <action name="action1280x720">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1280x720</string>
   </property>
  </action>
  <action name="action1366x720">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1366x768</string>
   </property>
  </action>
  <action name="action1920x1080">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>1920x1080</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="actionMain_Menu">
   <property name="text">
    <string>Main Menu</string>
   </property>
   <property name="shortcut">
    <string>Shift+M</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
